creator-node/src/index.ts

```diff
@@ -28,6 +28,8 @@ const verifyDBConnection = async () => {
 
 const runDBMigrations = async () => {
   try {
+    logger.info('Clearing running db queries...')
+    clearRunningQueries()
```

Comment:
nit - would be more readable if called directly inside `startApp`, but not worth bc of the "Connect" part - could call this inside `connectToDBAndRunMigrations` instead but w/e prob not worth

Reply:
@SidSethi take a look at my most recent commit and let me know what you think

Reply:
perf